source/common/router/config_impl.cc

```diff
@@ -1382,7 +1382,8 @@ PathTemplateRouteEntryImpl::PathTemplateRouteEntryImpl(
     Server::Configuration::ServerFactoryContext& factory_context,
     ProtobufMessage::ValidationVisitor& validator)
     : RouteEntryImplBase(vhost, route, optional_http_filters, factory_context, validator),
-      path_template_(route.match().path_template()),
+      // TODO(silverstar194) Implement path template matcher
```

Comment:
the pattern is raise a unimplemented exception.

Reply:
changed